### PR TITLE
Release/steam

### PR DESCRIPTION
**Pull request by Fueredoriku**
The first iteration of the steam build. The goal of this iteration was to make a suitable foundation for the steam release.

Includes changes from:

- 7b
- 8a
- Miscellaneous work after sprint process was concluded. (Summer 2022)
- Grinding out feature-creep and polish for some reason (August - September 2022)

This PR will be merged when valve has approved initial build on steamworks

### CODE REVIEW

**Review by SondreHus — APPROVED** [review]
I ain't readin all that. I'm happy for you tho. Or sorry that happened.

**Review by TArchetype — APPROVED** [review]

**Review by jesper2k — APPROVED** [review]
I read through all 734117 lines and i see no problem.

The README is especially nice 👍

File: README.md [hunk 1]
```diff
@@ -1,60 +1,56 @@
...
-[jesper2k](https://github.com/jesper2k)
-
-[SondreHus](https://github.com/SondreHus)
-
-[TArchetype](https://github.com/TArchetype)
+* [ddabble](https://github.com/ddabble)
+* [Fueredoriku](https://github.com/Fueredoriku)
+* [jesper2k](https://github.com/jesper2k)
+* [SondreHus](https://github.com/SondreHus)
+* [TArchetype](https://github.com/TArchetype)
```
Comment (jesper2k) [inline README.md:42]: Legge til våre fulle navn her også?

Comment (Fueredoriku) [reply README.md:42]: Vi kan godt det for min del, men da må vi høre med folk om de er komfortable med det først :)

Comment (jesper2k) [reply README.md:42]: @ddabble @TArchetype @SondreHus er dere komfortable med at vi har navnene vår her?

Comment (ddabble) [reply README.md:42]: For min del vil jeg gjerne ha enten bare fornavnet mitt eller brukernavnet mitt 🙂 Kanskje spørre på slack, slik at de andre vi ikke har tagga her også kan svare 😊

**Review by TArchetype — APPROVED** [review]
Slack poll'en ga et ensidig svar, så godkjenner denne!

**Review by jesper2k — APPROVED** [review]
epic

**Comment by jesper2k** [comment]
200 commit long Frankenstein PR kekw

**Comment by Fueredoriku** [comment]
> 200 commit long Frankenstein PR kekw

Don't tempt me to add more 😏

**Pull request merged**